api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -130,8 +130,25 @@
     <T> Optional<T> getOptionalValue(String propertyName, Class<T> propertyType);
 
     /**
-     * Return all property names returned by any of the underlying {@linkplain ConfigSource configuration sources}.
-     * The order of the returned property names is unspecified.
+     * Returns a sequence of configuration property names. The order of the returned property names is unspecified.
```

Comment:
Sorry, missed this earlier.  Up to you whether you want to deal with it.  A sequence (definitionally) has order.  Perhaps you mean just "collection".

Reply:
Right but iterations are by definition a sequence, and thus they do have *an* order.  We just don't specify what the order is.